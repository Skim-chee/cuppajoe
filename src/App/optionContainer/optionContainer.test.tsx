import React from 'react';
import { shallow } from 'enzyme';
import OptionContainer from './optionContainer';
import toJson from 'enzyme-to-json';

describe('Header', () => {
	const wrapper = shallow(<OptionContainer />);

	it('render the component', () => {
		expect(wrapper.length).toEqual(1);
	});

	it('verifies no props', () => {
		expect(wrapper.instance().props).toEqual({});
	});

	it('verifies initial state', () => {
		expect(wrapper.state('optionOne')).toEqual(1);
		expect(wrapper.state('optionTwo')).toEqual(1);
	});

	it('matches the snapshot', () => {
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
