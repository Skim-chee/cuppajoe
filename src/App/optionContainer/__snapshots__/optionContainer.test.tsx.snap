// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches the snapshot 1`] = `
<div>
  <div>
    <OptionButton
      activeOption={1}
      feeling="chill"
      onOptionChange={[Function]}
    />
    <OptionButton
      activeOption={1}
      feeling="productive"
      onOptionChange={[Function]}
    />
  </div>
  <div>
    <OptionButton
      activeOption={1}
      feeling="stay"
      onOptionChange={[Function]}
    />
    <OptionButton
      activeOption={1}
      feeling="go"
      onOptionChange={[Function]}
    />
  </div>
</div>
`;
